examples/quickstart/jupyter-notebooks/Python_API_Tutorial.ipynb

```diff
@@ -0,0 +1,1281 @@
+       "   'version': '26.0.0-SNAPSHOT'},\n",
+       "  {'name': 'org.apache.druid.msq.guice.MSQServiceClientModule',\n",
+       "   'artifact': 'druid-multi-stage-query',\n",
+       "   'version': '26.0.0-SNAPSHOT'},\n",
+       "  {'name': 'org.apache.druid.msq.guice.MSQSqlModule',\n",
+       "   'artifact': 'druid-multi-stage-query',\n",
+       "   'version': '26.0.0-SNAPSHOT'},\n",
+       "  {'name': 'org.apache.druid.msq.guice.SqlTaskModule',\n",
+       "   'artifact': 'druid-multi-stage-query',\n",
+       "   'version': '26.0.0-SNAPSHOT'},\n",
+       "  {'name': 'org.apache.druid.server.lookup.namespace.NamespaceExtractionModule',\n",
+       "   'artifact': 'druid-lookups-cached-global',\n",
+       "   'version': '26.0.0-SNAPSHOT'},\n",
+       "  {'name': 'org.apache.druid.catalog.guice.CatalogCoordinatorModule',\n",
+       "   'artifact': 'druid-catalog',\n",
+       "   'version': '26.0.0-SNAPSHOT'},\n",
+       "  {'name': 'org.apache.druid.catalog.guice.CatalogBrokerModule',\n",
+       "   'artifact': 'druid-catalog',\n",
+       "   'version': '26.0.0-SNAPSHOT'}],\n",
+       " 'memory': {'maxMemory': 134217728,\n",
+       "  'totalMemory': 134217728,\n",
+       "  'freeMemory': 80642696,\n",
+       "  'usedMemory': 53575032,\n",
+       "  'directMemory': 134217728}}"
+      ]
+     },
+     "execution_count": 53,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "rest_client.get_json('/status')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "837e08b0",
+   "metadata": {},
+   "source": [
+    "A quick comparison of the three approaches (Requests, REST client, Python client):\n",
+    "\n",
+    "Status:\n",
+    "* Requests: `session.get(druid_host + '/status').json()`\n",
+    "* REST client: `rest_client.get_json('/status')`\n",
+    "* Status client: `status_client.status()`\n",
+    "\n",
+    "Health:\n",
+    "* Requests: `session.get(druid_host + '/status/health').json()`\n",
+    "* REST client: `rest_client.get_json('/status/health')`\n",
+    "* Status client: `status_client.is_healthy()`\n",
+    "\n",
+    "Ingest data:\n",
+    "* Requests: See the [REST tutorial](api_tutorial.ipynb)\n",
+    "* REST client: as the REST tutorial, but use `rest_client.post_json('/druid/v2/sql/task', sql_request)` and\n",
+    "  `rest_client.get_json(f\"/druid/indexer/v1/task/{ingestion_taskId}/status\")`\n",
+    "* SQL client: `sql_client.run_task(sql)`, also a form for a full SQL request.\n",
+    "\n",
+    "List datasources:\n",
+    "* Requests: `session.get(druid_host + '/druid/coordinator/v1/datasources').json()`\n",
+    "* REST client: `rest_client.get_json('/druid/coordinator/v1/datasources')`\n",
+    "* Datasources client: `ds_client.names()`\n",
+    "\n",
+    "Query data:\n",
+    "* Requests: `session.get(druid_host + '/druid/v2/sql', json=sql_request).json()`\n",
+    "* REST client: `rest_client.get_json('/druid/v2/sql', sql_request)`\n",
```

Comment:
@paul-rogers , to run a query, I think you would need to do a POST here & line 1034.

Reply:
and if there's a query with no headers etc, I had to use `post_only_json` to get it to work